certbot_dns_desec/dns_desec.py

```diff
@@ -4,17 +4,13 @@
 import time
 
 import requests
-import zope.interface
 from certbot import errors
-from certbot import interfaces
 from certbot.plugins import dns_common
 
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.DEBUG)
 
 
-@zope.interface.implementer(interfaces.IAuthenticator)
-@zope.interface.provider(interfaces.IPluginFactory)
```

Comment:
May these changes be responsible for this test failure?
https://github.com/desec-io/certbot-dns-desec/runs/4760133038?check_suite_focus=true

Reply:
No, 3.7 does not support f-strings ala `f"{my_var=}"` and flake8 gave a really unhelpful error message location (line 1 in the file containing that f-string).